tests/test_cli.py

```diff
@@ -76,7 +76,7 @@ def test_export_tiles(tmpdir, data):
     conn = sqlite3.connect(outputfile)
     cur = conn.cursor()
     cur.execute("select * from tiles")
-    assert len(cur.fetchall()) == 6
+    assert len(cur.fetchall()) == 5
```

Comment:
@jdmcbr I had to update the test expectations because one of the tiles was empty and is now skipped :)